### ISSUE TITLE
Hook up *1, *2, *3, *3 in REPL

### ISSUE THREAD

**Issue by borkdude** [babashka/nbb#85]

**Comment by zoren** [comment]
Also `*e` I get:
```
SyntaxError: Invalid regular expression: /*e/: Nothing to repeat
    at new RegExp (<anonymous>)
    at $APP.Kk (file:///opt/homebrew/lib/node_modules/nbb/lib/nbb_core.js:150:174)
    at file:///opt/homebrew/lib/node_modules/nbb/lib/nbb_api.js:20:315
    at file:///opt/homebrew/lib/node_modules/nbb/lib/nbb_core.js:959:214
    at file:///opt/homebrew/lib/node_modules/nbb/lib/nbb_core.js:959:227
    at Yg (file:///opt/homebrew/lib/node_modules/nbb/lib/nbb_core.js:91:468)
    at $APP.e.ia (file:///opt/homebrew/lib/node_modules/nbb/lib/nbb_core.js:890:22)
    at $APP.w (file:///opt/homebrew/lib/node_modules/nbb/lib/nbb_core.js:62:94)
    at file:///opt/homebrew/lib/node_modules/nbb/lib/nbb_core.js:902:70
    at Yg (file:///opt/homebrew/lib/node_modules/nbb/lib/nbb_core.js:91:468)
```

**Comment by borkdude** [comment]
I'll fix this hopefully this week

**Comment by borkdude** [comment]
Now released as 0.3.96
